rosco-web/config/rosco.yml

```diff
@@ -171,6 +187,18 @@ azure:
         version: 7.1.20150731
         osType: Linux
         packageType: rpm
+    - baseImage:
+        id: windows-2012-r2
+        shortDescription: 2012 R2
+        detailedDescription: Windows Server 2012 R2 Datacenter
+        publisher: MicrosoftWindowsServer
+        offer: WindowsServer
+        sku: 2012-R2-Datacenter
+        version: 4.0.20170111
+        osType: Windows
```

Comment:
The `windows` up above is lower-case, but this one is capitalized. I don't have a preference so long as each style works. But we should just pick one.